bundle/manifests/bpfman.io_fentryprograms.yaml

```diff
@@ -149,11 +149,13 @@ spec:
                           items:
                             type: string
                           type: array
+                          x-kubernetes-list-type: atomic
```

Comment:
Did you do anything to cause this to change, or did it just happen when you regenerated the crds?

Reply:
Possibly due to an update in the tools.  I think that's the default anyway.